### DISCUSSION TITLE
Steam Deck Installation

### DISCUSSION THREAD

**Discussion by MadogPL** [sezanzeb/input-remapper#587] [Q&A]
Hallo!

Well, I'm new to Linux OS and it's versions and limitations.
From what I understand Steam OS is a Arch architecture.
It uses the flatpak as package installation. 

And none of the files in here are flatpak.
To install on Arch I need to run pacaur command (?) on konsole, but to run on console I need to "open up" the steamOS to other means by 

`sudo steamos-readonly disable`

(https://help.steampowered.com/en/faqs/view/671A-4453-E8D2-323C)

and setup sudo password (as deck ships with no password), to install pacaur to be able to install Input Remapper :)

OR... maybe I'm missing something and there is easier way to do it. Like... dunno... making the package into a flatpak.

**Comment by yngmjgsd** [comment]
There is already a discussion about this #21, and so far it needs someone with Flatpak experience to continue this kind of activity.
